apis/cache/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -16,6 +16,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// DO NOT EDIT.
```

Comment:
WIll having this as part of `boilerplate.txt` cause `// DO NOT EDIT.` to appear twice whenever we move off of your fork of `controller-tools`? Do we have an issue tracking that eventual bump? If so, it might be useful to add this as a sub-point.

Also, did this give you errors with `golangci-lint`? (i.e. can you run `make reviewable` successfully without adding `// DO NOT EDIT.`?)

Reply:
> WIll having this as part of boilerplate.txt cause `// DO NOT EDIT.` to appear twice whenever we move off of your fork of controller-tools?

I don't think anything will change between the fork/original for this matter.

> can you run make reviewable successfully without adding `// DO NOT EDIT.`?

No, I cannot. It gives the error above. `DO NOT EDIT` lets the linter skip the file. We had another problem with that during kubebuilder v2 migration (generated files were not goimports-ed). I think skipping linting check on them makes sense since it's just machine-code.

Reply:
Hmm when I run `make reviewable` on master it is passing for me, and it must have passed on the last commit that was merged. Was there a change in this PR that would cause the linting to fail?

Reply:
It looks like #704 will be addressing this. Is that correct?

Reply:
Alright, they fixed it in `v0.2.0`, we don't need this anymore. https://github.com/crossplaneio/crossplane/pull/704